codewars9.py

```diff
@@ -0,0 +1,22 @@
+# https://www.codewars.com/kata/54e6533c92449cc251001667/train/python
+# I misunderstood the assignment and wrote code which only prints the first occurence of a character in a string.
+# I will repair it later.
+
+def unique_in_order(iterable):
+    iterable_new = []
+    iterable += " "
+    for index, x in enumerate(iterable):
+        if index in range(len(iterable) - 1):
+            if iterable.count(x) > 1:
+                if iterable[index + 1] == iterable[index]:
+                    continue
+                else:
+                    iterable_new.append(x)
+            else:
+                iterable_new.append(x)
+        else:
+            break
+    iterable = iterable_new
+    return iterable
```

Comment:
This exercise works just fine and passes all the test, congratulations! 

Just one small tip: You don't need to assign `iterable_new` to `iterable` variable. You can simply return `iterable_new`. This is just small remark for the future - you can return any variable, doesn't have to be the one that is passed with the function :)